addons/account/i18n/account.pot

```diff
@@ -9480,6 +9480,17 @@ msgstr ""
 msgid "PDF File"
 msgstr ""
 
+#. module: account
```

Comment:
I don't think you need to add translations in master?

Reply:
This is only the `.pot` file. I though new terms needed to be added in the `.pot` file, but I might be wrong. I saw you also did that with your previous PEPPOL PR.

Reply:
I might be wrong, just judging based on the table from Ruben :laughing: it's a non-localization module and we're updating master now. I think the previous PR you're referring to was created in stable and fw-ported? :slightly_smiling_face: